fvm/fvm.go

```diff
@@ -242,3 +247,74 @@ func (vm *VirtualMachine) GetAccount(
 	}
 	return account, nil
 }
+
+// GetAccountBalance returns an account balance by address or an error if none exists.
+func (vm *VirtualMachine) GetAccountBalance(
```

Comment:
it doesn't look like `vm` is actually used in any of these, so we probably don't need to add them to the `VirtualMachine` interface. Instead, just make them standalong methods.